packages/@sanity/portable-text-editor/src/types/editor.ts

```diff
@@ -383,11 +383,9 @@ export interface BlockListItemRenderProps {
   editorElementRef: React.RefObject<HTMLElement>
   focused: boolean
   level: number
-  nextItem?: string
   path: Path
-  previousItem?: string
```

Comment:
The `nextItem` and `previousItem` was never meant to be there. Experimental types that accidentially got checked in previously.